src/main/java/com/commercetools/emailprocessor/model/ProjectConfiguration.java

```diff
@@ -0,0 +1,38 @@
+package com.commercetools.emailprocessor.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ProjectConfiguration {
+    private static final Logger LOG = LoggerFactory.getLogger(ProjectConfiguration.class);
+    private List<TenantConfiguration> tenants;
```

Comment:
the class must be immutable - make it `final`

Reply:
8777eb238deeda4af6dbb283c616c3a441dcdc31

Reply:
1fadcd0